go.mod

```diff
@@ -8,6 +8,8 @@ require (
 	github.com/go-kit/kit v0.9.0
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.1.4
+	github.com/gorilla/handlers v1.5.1
```

Comment:
You should be really mindful every time a new package is included into the codebase. Do we know how this package works? Have we vetted it in detail? Is it worth it to include two new packages just to enable Cors in the json-rpc package?